src/main/java/frc/robot/commands/drivingCommands/DriveAndLoadingAimCommand.java

```diff
@@ -12,6 +12,8 @@
     private Limelight limelight;
     private Controls driverControls;
 
+    private double slowSpeed = 0;
```

Comment:
This variable is only used in one method, and it's not retained between frames (times execute is called).  Variables like that should be local to the method using them.

Reply:
I think this is in-order to maintain the slow speed when we lose sight of the target

Reply:
Can we initialize the variable in the constructor rather than setting it to 0 when creating the class variable?

Reply:
@kushagharahi We need to come up with a coding standard

Reply:
So I'm getting that I should add it either in the constructor or the initialization method?

Reply:
You are constructing it in the right place [the private double slowSpeed; part] (Kusha showed me how I was wrong earlier) but it should be initialized in the constructor [slowSpeed = 0.0;]

Reply:
okay, I'll change that right now